library.properties

```diff
@@ -1,9 +1,10 @@
 name=Adafruit TensorFlow Lite
-version=1.2.1
+version=1.3.0
 author=Adafruit
 maintainer=Adafruit <info@adafruit.com>
 sentence=This is a library to abstract away TensorFlow Lite for Adafruit/Arcada boards
 paragraph=This is a library to abstract away TensorFlow Lite for Adafruit/Arcada boards
 category=Data Processing
 url=https://github.com/adafruit/Adafruit_TFLite
 architectures=*
+depends=Arduino_TensorFlowLite, Adafruit Arcada Library, Adafruit Zero PDM Library, Adafruit Unified Sensor, Adafruit NeoPixel, Adafruit ADT7410 Library, Adafruit GFX Library, Adafruit ST7735 and ST7789 Library, Adafruit ILI9341, Adafruit EPD, Adafruit SPIFlash, Adafruit LIS3DH, Adafruit ZeroTimer Library, Adafruit TouchScreen, Adafruit MSA301, Adafruit BusIO, Adafruit PixelDust, ArduinoJson, Adafruit ImageReader Library, Adafruit WavePlayer Library, Adafruit TinyUSB Library, SdFat - Adafruit Fork, Adafruit Circuit Playground, Audio - Adafruit Fork, https://github.com/adafruit/WiFiNINA.git
```

Comment:
only dep is  Adafruit Arcada Library, rest come for 'free' when you install Arcada

Reply:
@ladyada Ah, I did not know that. I'll remove the others.